lib/explorer/series.ex

```diff
@@ -3539,6 +3539,91 @@ defmodule Explorer.Series do
   @doc type: :aggregation
   def frequencies(series), do: apply_series(series, :frequencies)
 
+  @doc """
+  Bins values into discrete values.
+
+  Given a `bins` length of N, there will be N+1 categories.
+
+  ## Options
+
+    * `:labels` - The labels assigned to the bins. Given `bins` of
+      length N, `:labels` must be of length N+1. Defaults to the bin
+      bounds (e.g. `(-inf -1.0]`, `(-1.0, 1.0]`, `(1.0, inf]`)
+
+    * `:break_point_label` - The name given to the breakpoint column.
+      Defaults to `break_point`.
+
+    * `:category_label` - The name given to the category column.
+      Defaults to `category`.
+
+    * `:maintain_order` - The name given to the category column.
+      Defaults to `false`.
+
+  ## Examples
+
+      iex> s = Explorer.Series.from_list([1.0, 2.0, 3.0])
+      iex> Explorer.Series.cut(s, [1.5, 2.5])
+      #Explorer.DataFrame<
+        Polars[3 x 3]
+         float [1.0, 2.0, 3.0]
```

Comment:
```suggestion
        float [1.0, 2.0, 3.0]
```

Reply:
That makes the doctest fail. I think it's because it's a column named with an empty string due to [this](https://github.com/pola-rs/polars/blob/rs-0.29.0/polars/polars-algo/src/algo.rs#L122).

I'll rename it to `values`.